### ISSUE TITLE
Fix shields.io badge fallback text

### ISSUE THREAD

**Issue by collinbarrett** [collinbarrett/FilterLists#74]
If the images from shields.io fail to load, the "alt" text of the images should still show. Currently, something in the WordPress theme/plugin hides the "alt" text.

**Comment by collinbarrett** [comment]
Won't fix on current site, but will be sure to implement properly on v3 currently under construction.
